logger/logger_test.go

```diff
@@ -0,0 +1,25 @@
+package logger
+
+import (
+	"context"
+	"testing"
+
+	goLogger "github.com/databricks/databricks-sdk-go/logger"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTfLogger_Enabled(t *testing.T) {
```

Comment:
@mgyucht Regarding unit test — mocking the tflog will make code a bit complicated (mocking interface) and I think in our earlier discussion we didn’t want to go that way. In Go SDK logging_test we only check if a certain level is enabled (not asserting the actual logged message), 

Please let me know if we should keep things simple as is or go ahead with full unit tests, right now am only checking if logger is enabled by default and default logger is set.

Reply:
I don't think this is possible to unit test, unless tflog exposes some internal details. I don't think unit tests should be necessary here to be honest. Only other test you may want to have is that the logger is set when the provider is initialized.